deepfence_console/elastic-Dockerfile

```diff
@@ -4,10 +4,10 @@ LABEL deepfence.role=system
 
 ENV VERSION=7.10.2 \
     DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch"
-ENV ES_TARBAL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz" \
-    ES_TARBALL_ASC="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz.asc" \
-    EXPECTED_SHA_URL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz.sha512" \
-    ES_TARBALL_SHA="7b63237996569ccdc7c9d9e7cc097fcb23865396eddac30e5f02543484220d2fc70a7285b430877e5e76a5d8716d9682de9fc40d5e57a08f331e82011fc59756" \
+ENV ES_TARBAL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-linux-x86_64.tar.gz" \
```

Comment:
elasticsearch-oss still has the log4shell vulnerability. Only elasticsearch is patched.